<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>705</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginDlg
{

	background-image: url(:/img/img/bg.jpg);
}
</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="show_passward">
      <property name="styleSheet">
       <string notr="true">#show_passward{
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>show passward</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="status">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#login_button
{

	border-image: url(:/img/img/login.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Title">
      <property name="styleSheet">
       <string notr="true">#Title{
	border-image: url(:/img/img/neutalk.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passward_enter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_passward_enter{
	
	color: rgb(255, 255, 255);
	border-image: url(:/img/img/black.jpg);}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>enter passward here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_enter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_id_enter{
	color: rgb(255, 255, 255);
	border-image: url(:/img/img/black.jpg);}</string>
   </property>
   <property name="placeholderText">
    <string>enter mail here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>149</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#register_button{
	border-image: url(:/img/img/color.png);
	border-radius: 6px;
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Register here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/mail.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_passward">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/password.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_id</zorder>
  <zorder>label_passward</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>login_button</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>lineEdit_passward_enter</zorder>
  <zorder>lineEdit_id_enter</zorder>
  <zorder>register_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
